pkg/helm/helm_test.go

```diff
@@ -0,0 +1,244 @@
+package helm
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHelmRepositoryName(t *testing.T) {
+	tests := []struct {
+		name           string
+		repoURL        string
+		chart          string
+		expectedOutput string
+	}{
+		{
+			name:           "OCI",
+			repoURL:        "oci://registry-1.docker.io/bitnamicharts/apache",
+			chart:          "apache",
+			expectedOutput: "oci://registry-1.docker.io/bitnamicharts/apache",
+		},
+		{
+			name:           "HTTP",
+			repoURL:        "https://suse-edge.github.io/charts",
+			chart:          "metallb",
+			expectedOutput: "repo-metallb/metallb",
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			repoName := repositoryName(test.repoURL, test.chart)
+			assert.Equal(t, test.expectedOutput, repoName)
+		})
+	}
+}
+
+func TestAddRepoCommand(t *testing.T) {
+	var buf bytes.Buffer
+	cmd := addRepoCommand("apache", "oci://registry-1.docker.io/bitnamicharts/apache", &buf)
```

Comment:
Small nit: using `oci://registry-1.docker.io/bitnamicharts/apache` may be a bit misleading as `helm repo add repo-apache oci://registry-1.docker.io/bitnamicharts/apache` is invalid in practice.